docs/sql-ref-syntax-qry-explain.md

```diff
@@ -19,4 +19,126 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+
+The `EXPLAIN` statement provides the execution plan for the statement. 
+By default, `EXPLAIN` provides information about the physical plan.
+`EXPLAIN` does not support `DESCRIBE TABLE` statement.
```

Comment:
What does this statement mean? We can accept `EXPLAIN DESCRIBE TABLE`;
```
scala> sql("explain describe table gstest_empty").show(1, false)
+---------------------------------------------------------------------------------------------------+
|plan                                                                                               |
+---------------------------------------------------------------------------------------------------+
|== Physical Plan ==
Execute DescribeTableCommand
   +- DescribeTableCommand `gstest_empty`, false

|
+---------------------------------------------------------------------------------------------------+
```

Reply:
sorry I made a mistake while executing. Thank you. I corrected.